src/dataset.ts

```diff
@@ -37,6 +37,16 @@ export interface Dataset {
      * environment. This may change in the future.
      */
     environments?: Environment[];
+    
+    /**
+     * Visualization settings to be used when loading the dataset.
+     * 
+     * Basically just a dictionary: visualization options will have to be 
```

Comment:
This is not a dictionnary in JS/TypeScript, but an object with arbitrary keys. The `Map` type would be closer to a dictionary.